src/random_lcg.F90

```diff
@@ -53,6 +54,21 @@ function prn() result(pseudo_rn)
   end function prn
 
 !===============================================================================
+! GET_PRN_AHEAD generates a pseudo-random number which is 'n' times ahead from
+! current seed.
+!===============================================================================
+
+  function get_prn_ahead(n) result(pseudo_rn)
```

Comment:
I agree the names are pretty confusing as is. As a first suggestion how about the following:
- prn :arrow_right: prn
- get_prn_ahead :arrow_right: future_prn
- prn_skip :arrow_right: advance_prn_seed
- prn_skip_ahead :arrow_right: future_seed

I think those better describe what each of the procedures does. @smharper Open to your suggestions on this too.